http-conduit/http-conduit.cabal

```diff
@@ -42,6 +42,7 @@ library
 
     if flag(aeson)
       build-depends: aeson                 >= 0.8
+                   , attoparsec-aeson      >= 2.2
```

Comment:
This needs to be lowered to allow version 2.1 which is the empty package compatible with older aesons.